portfolio/src/main/webapp/js/main.js

```diff
@@ -11,18 +11,21 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+$(document).ready(function(){
+  $('.header').height($(window).height());
+})
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['My favorite dessert is ricotta cheesecake.', 'It is on my bucket list to see the Aurora Borelis.', 'My favorite place that I have visited is Switzerland'];
```

Comment:
nit: can we break this line into multiple lines for better readability?

Reply:
fixed!